to-json-schema.js

```diff
@@ -21,11 +21,30 @@ const typeMappings = {
   uuid: 'string',
 };
 
-const commonAttributesMapping = (avroDefinition, jsonSchema) => {
+const commonAttributesMapping = (avroDefinition, jsonSchema, isTopLevel) => {
   if (avroDefinition.doc) jsonSchema.description = avroDefinition.doc;
   if (avroDefinition.default !== undefined) jsonSchema.default = avroDefinition.default;
+
+  const fullyQualifiedName = getFullyQualifiedName(avroDefinition);
+  if (isTopLevel && fullyQualifiedName !== undefined) {
+    jsonSchema['x-parser-schema-id'] = fullyQualifiedName;
+  }
 };
 
+function getFullyQualifiedName(avroDefinition) {
+  let name;
+
+  if (avroDefinition.name) {
+    if (avroDefinition.namespace) {
+      name = `${avroDefinition.namespace}.${avroDefinition.name}`;
```

Comment:
can you provide in readme explanation why you followed `dot` approach here, to connect namespace with name? I'm interested as with this solution you later need a dedicated filter that will have to process this string first, I adds some complications imho. If in the end, we assign this value to `id`-like field, shouldn't we do camelCase here?

Reply:
There is a part in the Avro spec that says:

"In record, enum and fixed definitions, the fullname is determined in one of the following ways:
...
A name and namespace are both specified. For example, one might use "name": "X", "namespace": "org.foo" to indicate the fullname org.foo.X."

I expect that if people are using Avro, they understand now names and namespaces work.

https://avro.apache.org/docs/current/spec.html